### DISCUSSION TITLE
Is it just me or do the bots not actually check every 5 minutes as planned?

### DISCUSSION THREAD

**Discussion by FxllenCode** [upptime/upptime#193] [Q&A] [answered]
So this is stemming from an issue I opened the other day:

#188 

I was lucky enough to get a quick answer. (Thank you!)

Anyway, I changed the settings in the .uppertimerc.yml so graphics, ping, and other info should update every 5 minutes and commit to git history. However, I was displeased to find this isn't happening:

![image](https://user-images.githubusercontent.com/40151425/104109553-1de7b680-529d-11eb-9e2c-947164bee4e5.png)

Here is what the file looks like;

![image](https://user-images.githubusercontent.com/40151425/104109560-2d66ff80-529d-11eb-9353-681587bbc749.png)

Am I using this incorrectly? It's pretty important that this is checked as much as possible, as 15 minutes could make a large difference in this use case. Some help/more information would be greatly appreciated. Thanks! 🙂

**Comment by AnandChowdhary** [comment]
Can you share a link to your repository? I'll have a look!

A sidenote: You should also run "Update Template CI" once manually after updating your `.upptimerc.yml` to regenerate the workflows.

**Reply by FxllenCode** [reply]
https://github.com/FxllenCode/StatusPlus

Note a few things:

1. The current outage is artificial; just testing how the API responds to degraded performance.
2. It seems to be updating once an hour, which is no where near good enough. 
3. The use case for this is a bit weird; it's integrated with a gaming platform that has it's own status page, however it's often inaccurate and takes time to recognize an outage as it's manual. This detected an actual outage yesterday, but it was 6 minutes after the official status page, and the endpoint was already at 4000ms much before both of those. 

I've updated the template CI so that doesn't seem to be the problem.

**Reply by AnandChowdhary** [reply]
Can you try changing:

```yaml
  workflowSchedule:
  graphs: "*/5 * * * *"
```

to:

```yaml
workflowSchedule:
  graphs: "*/5 * * * *"
```

I think the indentation is important in YAML so it's not recognizing the KV pairs. Maybe just format the entire YAML file properly? `status-website:` is also twice.

**Reply by FxllenCode** [reply]
I've made the change; should find out if it's working in an hour or so. Thanks!

**Reply by FxllenCode** [reply]
So even with that change, it still does not make the changes every 5 minutes as planned. Very strange.

**Reply by Soneji** [reply]
Hey @FxllenCode I am having this issue too

I am suspecting GitHub might not be running */5 cron jobs every 5 minutes.

**Reply by AnandChowdhary** [reply]
@FxllenCode: The `workflowSchedule` is still not well-formatted: https://github.com/FxllenCode/StatusPlus/blob/master/.upptimerc.yml#L69.

**Reply by FxllenCode** [reply]
@AnandChowdhary I've made the change and haven't seen a difference... I will update if that changes but as of now it still isn't working as intended.

**Reply by AnandChowdhary** [reply]
No, it looks like it's updated as intended 😄 https://github.com/FxllenCode/StatusPlus/commit/8304e5c29a61e06e9b1a22037b5557666c8cb0bf

**Comment by AnandChowdhary** [comment, marked as answer]
I wrote a quick blog post with some additional information about this issue: https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working.

**Reply by Soneji** [reply]
Thanks for the article! Do you know of any other tools other than jenkins that can perform the same task? I want to use something as simple as possible

**Reply by Soneji** [reply]
Also is there any template for jenkins that matches the default settings of upptime?

**Reply by AnandChowdhary** [reply]
Not yet, but I'll try and set one up!

**Reply by ririko5834** [reply]
Hello, can you please send me example code how to run uptime workflow every 5 minutes using IFTTT?

**Reply by AnandChowdhary** [reply]
We now support using self-hosted runners, which is a possible solution to this: https://upptime.js.org/docs/configuration#self-hosted-runners. I'll also work on some docs for IFTTT or Jenkins.

**Reply by Soneji** [reply]
Surely self hosted runners doesnt make much sense since they use our infrastructure. One main benefit of upptime is that it is disconnected from our infrastructures, and therefore independant on them, and can report error if our infrastructures go down

**Reply by ririko5834** [reply]
Can you make guide for IFTTT or pipedream? I dont want self-hosted runner. @AnandChowdhary

**Reply by Soneji** [reply]
Hi @AnandChowdhary 
Is there any update on this?
